cogs/bot.py

```diff
@@ -188,7 +188,16 @@ async def on_guild_join(self, guild):
         )
         await channel.send(embed=embed)
 
-    async def determine_prefix(self, guild):
+    def case_insensitive_prefix(self, prefix, content):
+        match = re.match(f"^({prefix}).*", content, flags=re.I)
```

Comment:
prefix needs to be regex escaped

Reply:
cannot believe i glossed over this. definitely an issue haha!